test/tests/ghost-basics/run.sh

```diff
@@ -30,6 +30,8 @@ _request() {
 ghostVersion="$(docker inspect --format '{{range .Config.Env}}{{ . }}{{"\n"}}{{end}}' "$serverImage" | awk -F= '$1 == "GHOST_VERSION" { print $2 }')"
 case "$ghostVersion" in
 	0.*) _request GET '/ghost/' -I |tac|tac| grep -q '^Location: .*setup' ;;
-	*)   _request GET '/ghost/api/v0.1/authentication/setup/' |tac|tac| grep -q 'status":false' ;;
+	1.*)   _request GET '/ghost/api/v0.1/authentication/setup/' |tac|tac| grep -q 'status":false' ;;
+	2.*)   _request GET '/ghost/api/v2/admin/authentication/setup/' |tac|tac| grep -q 'status":false' ;;
+	3.*)   _request GET '/ghost/api/v3/admin/authentication/setup/' |tac|tac| grep -q 'status":false' ;;
```

Comment:
I'm a fan of the approach here, but I think we should still have some kind of fallback, even if it's just explicit failure (`false` or `exit 1`) to remind us at 4.x that something needs to be done here. :sweat_smile:

(Also the spacing after the `)` looks kind of odd -- was there a reason for the triple-space there?)

Reply:
makes sense, I'll fix that

Reply:
@tianon this should be fixed now 😄